### DISCUSSION TITLE
conda packages for python 3.9

### DISCUSSION THREAD

**Discussion by rmrmg** [rdkit/rdkit#3636] [Ideas] [answered]
In conda repo there are packages for relatively old python version. Is there any reason why newest version are not included?

**Comment by mwojcikowski** [comment, marked as answer]
Rafał, please use conda-forge as this is recommended repo now.

**Reply by greglandrum** [reply]
Just to reinforce that: I will still occasionally do builds in the rdkit channel, particularly of betas or nightlies, but the conda-forge binaries support more python versions, more operating systems, and are automatically produced, so that's a huge win.
